frontend/packages/noobaa-storage-plugin/src/utils.ts

```diff
@@ -26,3 +26,5 @@ export const getResiliencyProgress = (response: PrometheusResponse): number => {
   const progress = getGaugeValue(response);
   return Number(Number(progress).toFixed(1));
 };
+
+export const filterStorageClasses = (sc) => sc.provisioner.includes('noobaa.io/obc');
```

Comment:
can you add type for `sc` ?

Reply:
sc has a weird type it extends upon K8sResource.
https://github.com/openshift/console/blob/master/frontend/public/components/utils/storage-class-dropdown.tsx#L47-L62

Reply:
not needed anymore as you are using the one from `console-shared`